### PR TITLE
Define __repr__ on models

### LINKED ISSUE #7063: Implement __repr__ for models
Per the [design guidelines](https://azure.github.io/azure-sdk/python_design.html#models).

### PR DESCRIPTION
**Pull request by chlowell**
Closes #7063

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py [hunk 1]
```diff
@@ -1271,6 +1309,12 @@ def __init__(self, action_type, lifetime_percentage=None, days_before_expiry=Non
         self._days_before_expiry = days_before_expiry
         self._action_type = action_type
 
+    def __repr__(self):
+        # type () -> str
+        return "LifetitmeAction(action_type={}, lifetime_percentage={}, days_before_expiry={})".format(
```
Comment (bryevdv) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py]: LifetitmeAction -> LifetimeAction

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py [hunk 2]
```diff
@@ -1345,6 +1389,10 @@ def __init__(
         self._recovery_id = recovery_id
         self._scheduled_purge_date = scheduled_purge_date
 
+    def __repr__(self):
+        # type () -> str
+        return "<DeletedCertificatte [{}]>".format(self.id)[:1024]
```
Comment (bryevdv) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py]: DeletedCertificatte -> DeletedCertificate

**Pull request merged**